specification/ai/Azure.AI.Projects/datasets/routes.tsp

```diff
@@ -0,0 +1,55 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "../common/models.tsp";
+import "./models.tsp";
+import "../servicepatterns.tsp";
+
+using TypeSpec.Http;
+
+namespace Azure.AI.Projects;
+
+alias ListDatasetVersionsParameters = {
+  @doc("Top count of results, top count cannot be greater than the page size. If topCount > page size, results with be default page size count will be returned")
+  @query
+  top?: int32;
+
+  @doc("Continuation token for pagination.")
+  @query
+  skip?: string;
+
```

Comment:
Why do we need to support top and skip? This seems like something you provide when there is a very long list of items. I don't think that's the case with Foundry Projects. We don't have this in the current Project SDK, and we never heard a request for it.

Reply:
I left the same comment on the new PR. Resolving this one.